Mysql/select_items.sql

```diff
@@ -0,0 +1,139 @@
+#salt_sampled Level
+#total number of salt samples collected 
+
+SELECT
```

Comment:
I used this file to create the summaries view, so we don't need it anymore

Reply:
👍 deleting